src/domain/app/i18n/en.json

```diff
@@ -1252,6 +1253,15 @@
       "titleServiceHighlights": "Service highlights",
       "titleSupport": "Support and further development"
     },
+    "registrationInstructionsPage": {
+      "imageAltText1": "After publishing an event: got to the Registration section in Linked Events. Press the Add New button.",
```

Comment:
```suggestion
      "imageAltText1": "After publishing an event: go to the Registration section in Linked Events. Press the Add New button.",
```

Reply:
Good catch. This one was copy pasted from the original pdf-document. Now fixed